openmetadata-ui/src/main/resources/ui/src/components/AsyncSelectList/AsyncSelectList.tsx

```diff
@@ -222,6 +223,7 @@ const AsyncSelectList: FC<AsyncSelectListProps> = ({
         startWith={TAG_START_WITH.SOURCE_ICON}
         tag={tag}
         tagProps={tagProps}
+        tagType={tagType}
```

Comment:
Instead of passing the additional `tagType` prop, can we directly use the `tag.source` value here?

Reply:
The earlier issue which was happening was due to **data.source**,it returned **undefined** incase of already selected tag. This fix will work as expected now.